README.md

```diff
@@ -10,6 +10,20 @@ At present `stellar-core-backup` makes a few assumptions about the environment t
 
 AWS credentials can be exported as environment variables or permissions can be granted by an IAM instance role.
 
+## GPG Backup Signing and Verification
+
+Unless the user disables GPG signing and verification using the --no-verify flag, then the gpg checks will be run. The key details are loaded into the config file.
+
+The appropriate GPG signing keys also need to be in place. The public Stellar Backup key can be installed using the --getkey flag and the key-ID contained in the config/sample.yaml file.
```

Comment:
I think we should probably display the command to retrieve the key using `scb` otherwise users might find it easier to install using the manual example below.